master/reference/calicoctl/resources/profile.md

```diff
@@ -15,8 +15,9 @@ aliases are supported (all case insensitive): `profile`, `profiles`, `pro`, `pro
 
 ### Sample YAML
 
-The following sample profile allows all traffic from 10.0.0.0/16, except if the source 
-is within 10.244.0.0/24, or if the source is within the same profile.
+The following sample profile allows all traffic from endpoints that
+have the profile label set to profile1 (i.e. endpoints that reference this profile),
```

Comment:
can you add a `'` around `profile1` in indicate it's a name or make it a code by adding ` around it